res/css/views/auth/_InteractiveAuthEntryComponents.scss

```diff
@@ -55,4 +55,12 @@ limitations under the License.
 
 .mx_InteractiveAuthEntryComponents_termsPolicy {
     display: block;
-}
\ No newline at end of file
+}
+
+.mx_InteractiveAuthEntryComponents_passwordSection {
+    width: 300px;
+}
+
+.mx_InteractiveAuthEntryComponents_passwordSection input {
+    width: 100%;
```

Comment:
If you're interested, we could probably remove most of these `width: X` styles for all of the inputs inside `Field`s by doing https://github.com/vector-im/riot-web/issues/9344.